src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -476,10 +476,24 @@ public function __clone()
      */
     public function __toString()
     {
+        $protocol = '';
+        try {
+            $protocol = $this->server->get('SERVER_PROTOCOL');
+        } catch (\InvalidArgumentException $mallformedPath) {
+            trigger_error($mallformedPath->getMessage(), E_USER_ERROR);
+        }
+
+        $content = '';
+        try {
+            $content = $this->getContent();
+        } catch (\LogicException $usageRestrictions) {
+            trigger_error($usageRestrictions->getMessage(), E_USER_ERROR);
```

Comment:
This case can not happen also, because the exception is throw only when `true` given to getContent()